MBBSEmu/HostProcess/GlobalRoutines/IGlobalRoutine.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using MBBSEmu.Memory;
+using MBBSEmu.Module;
+using MBBSEmu.Session;
+
+namespace MBBSEmu.HostProcess.GlobalRoutines
+{
+    public interface IGlobalRoutine
+    {
+        bool ProcessCommand(byte[] command, ushort channelNumber, PointerDictionary<SessionBase> sessions,
```

Comment:
can you add some docs describing the return value at least?

Reply:
Added xml-doc for this method